hw_diag/utilities/network_watchdog.py

```diff
@@ -0,0 +1,81 @@
+import time
+import urllib.request
+from datetime import datetime, timedelta
+from hm_pyhelper.logger import get_logger
+from retry.api import retry_call
+from utilities.balena_supervisor import BalenaSupervisor
+
+logging = get_logger(__name__)
+
+RESTART_FILE_NAME = 'restarts.csv'
```

Comment:
This needs to be placed in a persistent storage path.

Reply:
yes, it is implemented now.
- https://github.com/NebraLtd/helium-miner-software/pull/459/files#diff-29d08d4ce056eee6ab4cba496b7a559f37150deb557009f8d3c4fd9fbd3912beR77
- `RESTART_FILE_NAME = '/var/data/restarts.csv'`